### DISCUSSION TITLE
Mssql 2019 as back end?

### DISCUSSION THREAD

**Discussion by bernardburns** [apache/airflow#21977] [Q&A]
Hello all, we are encountering issues using Mssql 2019 as our backend database for Airflow 2.2.3. The scheduler hangs when attempting to trigger a dag via the console and we get a deadlock when parallelism is greater than one. Thanks for your help!

**Comment by potiuk** [comment]
MSSQL is not yet supported  for 2.2.3 - it **might** get support in 2.3.0 but it is in development https://github.com/apache/airflow#requirements

**Reply by potiuk** [reply]
And we have implemented a lot of changes to make it work so the fact that it does not work (especially with deadlock) does not surprise me at all.

**Reply by bernardburns** [reply]
Thank you very much!  Just to let you know that the Apache site has misleading information concerning Mssql support in case if anyone else asks. https://airflow.apache.org/docs/apache-airflow/stable/howto/set-up-database.html

**Reply by potiuk** [reply]
RIGHT! THANKS! 

That was cherry-picked by mistake! https://github.com/apache/airflow/pull/21998 to fix it.

**Reply by potiuk** [reply]
Should be fixed in https://airflow.apache.org/docs/apache-airflow/stable/howto/set-up-database.html
